test/jest-e2e.json

```diff
@@ -2,7 +2,7 @@
   "moduleFileExtensions": ["js", "json", "ts"],
   "rootDir": "../",
   "testEnvironment": "node",
-  "testRegex": ".e2e-spec.ts$",
+  "testRegex": ".e2e.spec.ts$",
```

Comment:
any reason for renaming?

Reply:
I renamed files because otherwise it is not possible to start them from Jest Test Explorer. It uses settings from package.json to find tests and it is "testRegex": ".spec.ts$". If test has file name .e2e-spec.ts then it does not match the pattern and test is not executed. For the same reason I had to change rootDir and added testTimeout property.